.travis.yml

```diff
@@ -1,6 +1,5 @@
 sudo: required
 dist: trusty
-os: linux
```

Comment:
Well, I assume the goal was to provide `osx` builds one day...

Though, I am not sure on the eta of this and it can always be added back in later.